haystack/nodes/retriever/link_content.py

```diff
@@ -43,31 +55,67 @@ class LinkContentFetcher(BaseComponent):
 
     LinkContentFetcher supports the following content types:
     - HTML
+    - PDF
 
     """
 
     outgoing_edges = 1
 
+    USER_AGENT = f"haystack/LinkContentRetriever/{__version__}"
+
     REQUEST_HEADERS = {
         "accept": "*/*",
-        "User-Agent": f"haystack/LinkContentFetcher/{__version__}",
+        "User-Agent": USER_AGENT,
         "Accept-Language": "en-US,en;q=0.9,it;q=0.8,es;q=0.7",
         "referer": "https://www.google.com/",
     }
```

Comment:
I'm no Python guru, but I think If the constants are specific to the LinkContentFetcher class and are not intended to be used elsewhere, keeping them inside the class provides a clear namespace. This way, when someone refers to the constant, it's clearly associated with this specific class (e.g., LinkContentFetcher.USER_AGENT). What's amiss here is that these are, in fact, class variables, and users can change their values. I guess I could introduce read-only properties, but perhaps that's going overboard. I'm guessing since there's no built-in way to enforce immutability for constants, this current approach should be ok. LMK

Reply:
No need to enforce immutability, I was simply making sure there was no hidden mechanism that required them to be class properties. 

If they're not meant to be user facing anyway, you could move them out and prepend them with `_`. This way it will be obvious to people that they're not meant to be used or imported. Otherwise, by putting them in the class you end up making them more prominent, not less: that's because they will show up with the autocomplete of the class and may trick users into thinking this is an actual API of some sort.